tests/jest/non-root-package-json/__snapshots__/index.test.js.snap

```diff
@@ -2,8 +2,8 @@
 
 exports[`Dependency tree with non-root package.json should output warnings 1`] = `
 "a
-  Multiple versions of a found:
-    1.0.0 ./~/a
-    2.0.0 ./~/b/~/a
+[0m  Multiple versions of [0m[32m[1ma[22m[39m[37m found:[39m
+[37m[39m    [32m[1m1.0.0[22m[39m [37m[1m./~/a[22m[39m
+    [32m[1m2.0.0[22m[39m [37m[1m./~/b/~/a[22m[39m
```

Comment:
Are these snapshots being written how you would expect with this update? This is a little unreadable, are these special charters for colors/formatting?

Reply:
yeah, thats the color formatting for the terminal output.

Reply:
If you want to dig in deeper they're ansi escape codes https://en.wikipedia.org/wiki/ANSI_escape_code
I think at on point is was trying to track down those codes for some reason and they were really hard to google :)